### PR TITLE
Create intial page for `CS8355`

### PR DESCRIPTION
**Pull request by BartoszKlonowski**
This pull request introduces initial page for `CS8355` compilation error.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/language-reference/compiler-messages/cs8355.md](https://github.com/dotnet/docs/blob/d8feddbf3f48dcb36bcb8dd7b8869dc276cc4e7d/docs/csharp/language-reference/compiler-messages/cs8355.md) | [Compiler Error CS8355](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs8355?branch=pr-en-us-38699) |
| [docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md](https://github.com/dotnet/docs/blob/d8feddbf3f48dcb36bcb8dd7b8869dc276cc4e7d/docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md) | [Sorry, we don't have specifics on this C# error](https://review.learn.microsoft.com/en-us/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error?branch=pr-en-us-38699) |

### CODE REVIEW

File: docs/csharp/language-reference/compiler-messages/cs8355.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+f1_keywords:
+  - "CS8355"
+helpviewer_keywords:
+  - "CS8355"
+---
+# Compiler Error CS8355
+
+An in parameter cannot have the Out attribute.
+
+The parameter of a method can't be set as being passed by value, while informing interop marshaler to treat this parameter as passed by reference to the native code.
```
Comment (BillWagner) [inline docs/csharp/language-reference/compiler-messages/cs8355.md:14]: I'd reword this a bit. `in` parameters are (generally) passed by reference:

```suggestion
The parameter of a method can't be declared as read only, while informing interop marshaler to treat this parameter as writable by the native code.
```

**Review by BillWagner — COMMENTED** [review]
Thanks again @BartoszKlonowski 

I had one clarifying comment, and then it's ready to :shipit:

**Comment by BartoszKlonowski** [comment]
Closing and reopening to re-trigger the build.

**Review by BillWagner — APPROVED** [review]
This LGTM @BartoszKlonowski 

Thanks again for all the work you've done. I'll :shipit: now.
